### DISCUSSION TITLE
MQTT Version for bridge

### DISCUSSION THREAD

**Discussion by beckerben** [vernemq/vernemq#1865] [Q&A]
Is it not possible that we specify in the configuration the MQTT version to use for a bridge connection?

**Comment by ioolkos** [comment]
@beckerben Thanks, yes, that's correct. The bridge implements the 3.1.1 protocol version internally. It will forward MQTT v5 messages that are incoming on the originating broker, but loose the properties part of those messages.

Ideally, the bridge would implement both protocol versions, with a per-bridge protocol config option.

**Reply by beckerben** [reply]
Got it thanks, and I agree, ideally this is something that could be configured for the bridge specifically which protocol version to use. Thanks for the reply!
